receiver/bindplaneauditlogs/logs.go

```diff
@@ -0,0 +1,238 @@
+	logEvents, err := r.getLogs(ctx)
+	if err != nil {
+		return fmt.Errorf("getLogs: %w", err)
+	}
+	observedTime := pcommon.NewTimestampFromTime(time.Now())
+	logs := r.processLogEvents(observedTime, logEvents)
+
+	if !(logs.LogRecordCount() > 0) {
+		r.logger.Debug("no logs to process")
+		return nil
+	}
+
+	if err := r.consumer.ConsumeLogs(ctx, logs); err != nil {
+		return fmt.Errorf("consumeLogs: %w", err)
+	}
+
+	return nil
+}
+
+func (r *bindplaneAuditLogsReceiver) getLogs(ctx context.Context) ([]AuditLogEvent, error) {
+	var logs []AuditLogEvent
+	const timeout = 1 * time.Minute
+
+	reqURL := *r.cfg.bindplaneURL
+	reqURL.Path = "/v1/audit-events"
+
+	// Only add the since parameter if we have a lastTimestamp
+	if r.lastTimestamp != nil {
+		q := reqURL.Query()
+		q.Add("min-date", r.lastTimestamp.Format("20060102150405"))
+		reqURL.RawQuery = q.Encode()
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, timeout)
+	defer cancel()
+
+	req, err := http.NewRequestWithContext(ctx, "GET", reqURL.String(), nil)
+	if err != nil {
+		return nil, fmt.Errorf("newRequestWithContext: %w", err)
+	}
+
+	req.Header.Add("X-Bindplane-Api-Key", r.cfg.APIKey)
+
+	res, err := r.client.Do(req)
+	if err != nil {
+		return nil, fmt.Errorf("client.Do: %w", err)
+	}
+	defer res.Body.Close()
+
+	if res.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("non-200 response: %s", res.Status)
+	}
+
+	body, err := io.ReadAll(res.Body)
+	if err != nil {
+		return nil, fmt.Errorf("io.ReadAll: %w", err)
+	}
+
+	var response apiResponse
+	err = json.Unmarshal(body, &response)
+	if err != nil {
+		return nil, fmt.Errorf("unable to unmarshal log events: %w", err)
+	}
+
+	logs = response.AuditEvents
+
+	// Sort by timestamp (newest first)
+	sort.Slice(logs, func(i, j int) bool {
+		return logs[i].Timestamp.After(*logs[j].Timestamp)
+	})
+
+	if len(logs) > 0 {
+		// first log has the newest timestamp
+		r.lastTimestamp = logs[0].Timestamp
+	}
+
+	return logs, nil
+}
+
+func (r *bindplaneAuditLogsReceiver) processLogEvents(observedTime pcommon.Timestamp, logEvents []AuditLogEvent) plog.Logs {
+	logs := plog.NewLogs()
+	resourceLogs := logs.ResourceLogs().AppendEmpty()
+
+	// Add resource attributes
+	resourceAttrs := resourceLogs.Resource().Attributes()
+	resourceAttrs.PutStr("bindplane_url", r.cfg.Endpoint)
+
+	scopeLogs := resourceLogs.ScopeLogs().AppendEmpty()
+
+	for _, logEvent := range logEvents {
+		logRecord := scopeLogs.LogRecords().AppendEmpty()
+
+		// Set timestamps
+		logRecord.SetObservedTimestamp(observedTime)
+		if logEvent.Timestamp != nil {
+			logRecord.SetTimestamp(pcommon.NewTimestampFromTime(*logEvent.Timestamp))
+		}
+
+		// Set body as description
+		body := logRecord.Body()
+		if logEvent.Description != "" {
+			body.SetStr(logEvent.Description)
+		} else {
+			body.SetStr(fmt.Sprintf("Audit log event - Action: %s, Resource: %s, User: %s", logEvent.Action, logEvent.ResourceName, logEvent.User))
+		}
```

Comment:
Since there's still a chance old audit events have an empty description we should still check
```suggestion
		// Set body as description
		if logEvent.Description != "" {
			logRecord.Body().SetStr(logEvent.Description)
		}
```